internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -497,16 +497,22 @@ func (s *Server) parseArtifactPublishedEvent(
 	}
 
 	dbArtifact, err := s.store.UpsertArtifact(ctx, db.UpsertArtifactParams{
-		RepositoryID:       dbrepo.ID,
+		RepositoryID: uuid.NullUUID{
+			UUID:  dbrepo.ID,
+			Valid: true,
+		},
 		ArtifactName:       tempArtifact.GetName(),
 		ArtifactType:       tempArtifact.GetTypeLower(),
 		ArtifactVisibility: tempArtifact.Visibility,
+		ProjectID:          dbrepo.ProjectID,
+		ProviderID:         dbrepo.ProviderID,
+		ProviderName:       dbrepo.Provider,
```

Comment:
I'm getting:
```
{"level":"debug","Timestamp":1712687249228835000,"message":"Error parsing github webhook message: error upserting artifact: pq: there is no unique or exclusion constraint matching the ON CONFLICT specification"}
```
when trying to upsert an artifact with this PR.

My database is on version 50.